app/views/cookbooks/_manage_cookbook.html.erb

```diff
@@ -61,7 +61,7 @@
             <%= f.hidden_field :replacement, class: 'cookbook-deprecate', 'data-url' => deprecate_search_cookbook_path(cookbook) %>
           </div>
           <div class="small-3 columns">
-            <%= f.submit 'Deprecate', class: 'button radius postfix submit-deprecation', disabled: true %>
+            <%= f.submit 'Deprecate', class: 'button radius postfix submit-deprecation' %>
```

Comment:
Related to the JS removal, this leaves the "Deprecate" button enabled for the form to submit empty text.